### ISSUE TITLE
localOption (HideSuccesses True)   doesn't appear to work

### ISSUE THREAD

**Issue by dustin** [feuerbach/tasty#231]
I've got a test that has about 4,000 cases I'm trying to fix.  It's a generated subsuite and I'd really like to not see it unless something is broken.

`-ta --hide-successes` works but just prefixing this particular bit with `localOption (HideSuccesses True)` doesn't.

e.g.:

    testPartialTable :: [TestTree]
    testPartialTable = undefined

    [...]

    localOption (HideSuccesses True) $ testGroup "partial table" testPartialTable,

**Comment by feuerbach** [comment]
Yes, HideSuccesses is a global option at the moment. It's possible to make it local, though that'd be a bit involved. Feel free to give it a try, the relevant code is in core/Test/Tasty/Ingredients/ConsoleReporter.hs.

**Comment by quchen** [comment]
Just stumbled upon this, the option still does not work (tasty 1.4.2.1).
